jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/SelectedLogGroup.kt

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import com.intellij.util.xmlb.annotations.Tag
+import java.time.Instant
+import java.util.Date
+
+@Tag("SelectedLogGroups")
+data class SelectedLogGroups(
+    var logGroups: String? = null
+)
+
+data class QueryDetails(
+    val logGroupName: String,
```

Comment:
Is this the name of the initially selected log group? Shouldn't we be able to get to this menu from not a log group as well (in the future)?

Reply:
How would you represent multiple log groups? You may want to turn this into an array.

Reply:
Converted to an array